src/motion/cg_optimizer.F

```diff
@@ -198,14 +290,33 @@ RECURSIVE SUBROUTINE cp_cg_main(force_env, x0, gopt_param, output_unit, globenv,
          ![NB] consistent energies and forces not required for CG, but some line minimizers might set it
          save_consistent_energy_force = gopt_env%require_consistent_energy_force
          gopt_env%require_consistent_energy_force = .FALSE.
-         CALL cp_eval_at(gopt_env, x0, opt_energy, xi, gopt_env%force_env%para_env%mepos, &
-                         .FALSE., gopt_env%force_env%para_env)
+
+         ! Symmetrize coordinates and forces
+         IF (keep_sg) THEN
+           CALL cp_eval_at(gopt_env, x0, opt_energy, xi, gopt_env%force_env%para_env%mepos, &
+                           .FALSE., gopt_env%force_env%para_env, spgr=spgr)
+         ELSE
+           CALL cp_eval_at(gopt_env, x0, opt_energy, xi, gopt_env%force_env%para_env%mepos, &
+                           .FALSE., gopt_env%force_env%para_env)
+         END IF
          gopt_env%require_consistent_energy_force = save_consistent_energy_force
 
+         ! Symmetrize coordinates and forces
+         IF (keep_sg) THEN
+            CALL spgr_apply_rotations_force(spgr, xi)
+         END IF
+
          ! Get Conjugate Directions:  updates the searching direction (h)
          wildcard = "   CG"
          CALL get_conjugate_direction(gopt_env, Fletcher_Reeves, g, xi, h)
 
+         ! Symmetrize coordinates and forces
+         IF (keep_sg) THEN
+            CALL spgr_apply_rotations_force(spgr, g)
+            CALL spgr_apply_rotations_force(spgr, xi)
```

Comment:
I think the `CALL spgr_apply_rotations_force(spgr, xi)` could be dropped here, right?